docs/Installation/Installation.md

```diff
@@ -51,6 +51,7 @@ all of these dependencies.
 * [matplotlib](https://matplotlib.org/)
 * [PyYAML](https://pyyaml.org/)
 * [h5py](https://www.h5py.org)
+* [Arpack](https://github.com/opencollab/arpack-ng)
```

Comment:
Arpack will need to be installed on clusters. @kidder @wthrowe Do you have suggestions for doing that?

Reply:
And third: Please add a few notes on the new required dependency and some brief instructions to the "upgrade instructions" section of the PR description. This will help people upgrade their build environments once this PR is merged.